Makefile

```diff
@@ -9,7 +9,8 @@ type:
 
 .PHONY: format
 format:
-	@ruff format src/ test/
+	@find src -name '*.py' | xargs autopep8 --in-place
+	@find test -name '*.py' | xargs autopep8 --in-place
```

Comment:
I ran this against simply doing
```
autopep8 --in-place --recursive .
```
and it's a tiny bit faster (milliseconds).  I wonder if it'll make a difference in the long run when our file numbers grow.

Reply:
Actually, the issue seems to be the double call to find.
```
@find src test -name '*.py' | xargs autopep8 --in-place
```
is also a bit quicker.

Reply:
Went for the second option in a new commit :)